scheduler/client.c

```diff
@@ -2109,18 +2109,13 @@ cupsdSendHeader(
     }
     else if (auth_type == CUPSD_AUTH_NEGOTIATE)
     {
-#if defined(SO_PEERCRED) && defined(AF_LOCAL)
```

Comment:
I wasn't aware how these changes can affect macOS.

The 'PeerCred' mech will still be in WWW-Authenticate string together with 'Negotiate' and 'Local', added just below.

Then the `cups_local_auth` will work also for macOS because of this check:

```
 if (http->hostaddr->addr.sa_family == AF_LOCAL &&
      !getenv("GATEWAY_INTERFACE") &&	/* Not via CGI programs... */
      cups_auth_find(www_auth, "PeerCred"))
  {
```